cornflow-server/cornflow/shared/utils.py

```diff
@@ -3,12 +3,14 @@
 """
 import hashlib
 import json
+from cornflow_core.shared import database
+from cornflow_core.shared import password_crypt
 
-from flask_sqlalchemy import SQLAlchemy
-from flask_bcrypt import Bcrypt
+# from flask_sqlalchemy import SQLAlchemy
+# from flask_bcrypt import Bcrypt
 
-db = SQLAlchemy()
-bcrypt = Bcrypt()
+db = database
+bcrypt = password_crypt
```

Comment:
I had the impression that in most files you imported database and password_crypt directly from cornflow_core, so I don't think they need to be defined here anymore

Reply:
It should not be needed, but I have not changed all the imports just yet